exercises/rps_pattern_matching.livemd

```diff
@@ -90,8 +90,13 @@ defmodule RockPaperScissorsPatternMatching do
     iex> RockPaperScissorsPatternMatching.play(:rock, :paper)
     "paper beats rock!"
 
+  <<<<<<< HEAD
```

Comment:
No big deal, but these are unresolved merge conflicts. No action is necessary, but it's good to be aware that these accidentally got committed.